pom.xml

```diff
@@ -0,0 +1,64 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>portfolio</artifactId>
+  <version>1</version>
+  <packaging>war</packaging>
+
+  <properties>
+    <!-- Java 8. Potentially subject to change.-->
```

Comment:
Nick didn't comment on whether we should use Java 8 or 11, I asked him in the chat and he didn't say anything so I don't think he cares. App Engine fully supports (and recommends) Java 11 in the configuration that we're going to be using it in, so could we upgrade it?

Reply:
I'm down. Grace?

Reply:
@gracetian6

Reply:
yes lets upgrade!